tfe/resource_tfe_team.go

```diff
@@ -22,14 +23,38 @@ func resourceTFETeam() *schema.Resource {
 			"name": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
 			},
 
 			"organization": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
+
+			"organization_access": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				MinItems: 1,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"manage_policies": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+
+						"manage_workspaces": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+
+						"manage_vcs_settings": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
```

Comment:
ℹ️ I think it makes sense to allow `organization_access` to be completely optional (all these would default to `false` on a team creation, then) but IF you specify it you must specify all three settings; I think it's weird to not be explicit as you're _partially_ depending on the server's default. Feels like you should go all-in or not. Pointing this out though, to ask if this is reasonable or weird given usual conventions.

Reply:
Ah, we cross posted :wink: I don't see why that would be required? If I only want to give `manage_workspaces` access, then why should I need to fill in the other values as well?

To be clear, you do not depend on the servers default as the schema can define a default that would also be in the docs.

Reply:
> To be clear, you do not depend on the servers default as the schema can define a default that would also be in the docs.

Yeah that's what made me realize I really shouldn't care 😄Changed!